frontend/src/components/BookApp.tsx

```diff
@@ -27,10 +29,31 @@ const [bookList,setBookList]=useState<BookData[]>([])
     }
  }
 
+ function getBookByKeyword(keyword: string){
+    setBookList(filteredBookByTitle(keyword))
+ }
+
+ const filteredBookByTitle = (word: string) => {
+    return bookList.filter(book => {
+        return book.title.toLowerCase().includes(word.toLowerCase())
+    })
+ }
+
+    function getBookByISBN(isbn: string){
+        setBookList(filteredBookByISBN(isbn))
+    }
+
+    const filteredBookByISBN = (isbn: string) => {
+        return bookList.filter(book => {
+            return book.isbn.toLowerCase().includes(isbn.toLowerCase())
+        })
+    }
```

Comment:
Hier verwendet ihr eure schönen neuen Rest Endpunkte gar nicht, sondern filtert über die filter() Methode in TypeScript :) 
Müsst ihr die Endpunkte hier nicht noch anbinden?